tools/actions/composites/bot/action.yml

```diff
@@ -0,0 +1,80 @@
+name: "Bot"
+description: "Build and run the bot tests"
+inputs:
+  SEED:
+    description: "seed to use"
+    required: true
+  BOT_FILTER_FAMILY:
+    description: "family to filter on"
+    required: false
+  SLACK_CHANNEL:
+    description: "slack channel to report on"
+    required: false
+  SLACK_API_TOKEN:
+    description: "slack api token"
+    required: false
+  SHOW_LEGACY_NEW_ACCOUNT:
+    description: "SHOW_LEGACY_NEW_ACCOUNT value"
+    required: false
+  EXPLORER:
+    description: "EXPLORER value"
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - uses: pnpm/action-setup@v2.0.1
+      with:
+        version: latest
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16.x
+        cache: pnpm
+        cache-dependency-path: "**/pnpm-lock.yaml"
+    - name: pull docker image
+      run: docker pull ghcr.io/ledgerhq/speculos
+      shell: bash
+    - name: install and build
+      run: |
+        pnpm i --filter="live-cli..." --filter="ledger-live" --frozen-lockfile
+        pnpm build:llc
+      shell: bash
+    - name: bot logic
```

Comment:
Because we are not using jest, do we need a timeout here in case of something went wrong ?

Reply:
timeout is not possible to be set in a composite. this is why it's on the "usage" side , and it has been set to 120 :)

https://github.com/actions/runner/blob/main/docs/adrs/0549-composite-run-steps.md#timeout-minutes